### PR TITLE
IBX-1868: Documentation for migration has been supplemented

### PR DESCRIPTION
**Pull request by mateuszdebinski**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [IBX-1868](https://issues.ibexa.co/browse/IBX-1868)
| Versions      | v3, v4

### CODE REVIEW

File: docs/guide/data_migration/data_migration.md [hunk 1]
```diff
@@ -17,4 +17,9 @@ between installations by using the migration command.
     If you are migrating your data either with [`kaliop-uk/ezmigrationbundle`](https://github.com/kaliop-uk/ezmigrationbundle) or [`ezsystems/ezmigrationbundle`](https://github.com/ezsystems/EzMigrationBundle), do not install the [`tanoconsulting/ezmigrationbundle2`](https://github.com/tanoconsulting/ezmigrationbundle2) package, or your application will stop working due to multiple duplicated classes.
     
     As of v3.3.3, the `ezmigrationbundle` package has been removed to mitigate this issue. 
-    It is recommended that you use the default `ibexa/migrations` package to migrate your data.    
+    It is recommended that you use the default `ibexa/migrations` package to migrate your data. 
+
+
+### Example of using migration
+
+You can find examples of using migrations with actions in the folder **vendor/ibexa/migrations/tests/bundle/Command/MigrateCommand/migrate-command-fixtures**
```
Comment (DominikaK) [inline docs/guide/data_migration/data_migration.md:25]:
```suggestion
```

Given we don't mention actions in this page yet, I'd move it to the dedicated page.

File: docs/guide/data_migration/data_migration_actions.md [hunk 2]
```diff
@@ -57,3 +58,101 @@ The following migration actions are available out of the box:
 In contrast with Kaliop migrations, actions provide you with ability to perform additional operations and extend
 the migration functionality. 
 See [creating your own Actions](create_migration_action.md).
+
+## Examples of how to use actions
+
```
Comment (DominikaK) [inline docs/guide/data_migration/data_migration_actions.md:63]:
```suggestion

You can find examples of using migrations with actions in your project's 
`vendor/ibexa/migrations/tests/bundle/Command/MigrateCommand/migrate-command-fixtures` folder.

```

File: docs/guide/data_migration/data_migration_actions.md [hunk 3]
```diff
@@ -57,3 +58,101 @@ The following migration actions are available out of the box:
 In contrast with Kaliop migrations, actions provide you with ability to perform additional operations and extend
 the migration functionality. 
 See [creating your own Actions](create_migration_action.md).
+
+## Examples of how to use actions
```
Comment (DominikaK) [inline docs/guide/data_migration/data_migration_actions.md:62]:
```suggestion
## Action usage examples
```

File: docs/guide/data_migration/data_migration_actions.md [hunk 4]
```diff
@@ -57,3 +58,101 @@ The following migration actions are available out of the box:
...
+
+### Actions for Content
```
Comment (DominikaK) [inline docs/guide/data_migration/data_migration_actions.md:64]:
```suggestion
### Content
```
```suggestion
### Actions for Content
```

File: docs/guide/data_migration/data_migration_actions.md [hunk 5]
```diff
@@ -57,3 +58,101 @@ The following migration actions are available out of the box:
...
+
+mode: Update
+``` yaml
+    actions:
+        - { action: assign_parent_location, value: 2 }
+        - { action: assign_section, id: 4 }
+        - { action: assign_section, identifier: 'media' }
+```
+
+### Actions for Content Type
```
Comment (DominikaK) [inline docs/guide/data_migration/data_migration_actions.md:81]:
```suggestion
### Content Types
```

File: docs/guide/data_migration/data_migration_actions.md [hunk 6]
```diff
@@ -57,3 +58,101 @@ The following migration actions are available out of the box:
...
+mode: Update
+``` yaml
+    actions:
+        - { action: assign_content_type_group, value: 'Media' }
+        - { action: unassign_content_type_group, value: 'Content' }
+        - { action: remove_field_by_identifier, value: 'short_title' }
+        - { action: remove_drafts, value: null }
+```
+
+### Actions for Role
```
Comment (DominikaK) [inline docs/guide/data_migration/data_migration_actions.md:98]:
```suggestion
### Roles
```

File: docs/guide/data_migration/data_migration_actions.md [hunk 7]
```diff
@@ -57,3 +58,101 @@ The following migration actions are available out of the box:
...
+            id: 42
+        -
+            action: assign_role_to_user
+            email: 'mail@invalid.c'
+        -
+            action: assign_role_to_user
+            login: foo
+```
+
+### Actions for User
```
Comment (DominikaK) [inline docs/guide/data_migration/data_migration_actions.md:120]:
```suggestion
### Users

File: docs/guide/data_migration/data_migration_actions.md [hunk 8]
```diff
@@ -57,3 +58,101 @@ The following migration actions are available out of the box:
...
+        -   
+            action: assign_user_to_role
+            id: 2
+            limitation:
+                type: Section
+                values:
+                    - 1
+```
+
+### Actions for User Group
```
Comment (DominikaK) [inline docs/guide/data_migration/data_migration_actions.md:140]:
```suggestion
### User Groups
```

**Review by DominikaK — CHANGES_REQUESTED** [review]

**Review by DominikaK — APPROVED** [review]

**Review by Steveb-p — APPROVED** [review]

**Pull request merged**